crates/backend/src/ast.rs

```diff
@@ -150,13 +155,29 @@ impl Program {
                     }
                     _ => {}
                 }
+                let comments: Vec<String> = f.attrs
+                                .iter()
+                                .filter_map(|a| {
+                                    if a.style == AttrStyle::Outer {
+                                        Some(
+                                            a.tts.clone().into_iter().filter_map(|t| match t {
+                                                TokenTree::Literal(lit) => Some(format!("{}", lit)),
+                                                _ => None,
+                                            })
+                                        )
+                                    } else {
+                                        None
+                                    }
+                                })
+                                .fold(vec![], |mut acc, a| {acc.extend(a); acc});
```

Comment:
This should be a call to `extract_doc_comments`, right?

Reply:
yep, this has been updated